### ISSUE TITLE
Option to increase Xray graph scan timeout

### ISSUE THREAD

**Issue by colourspectrum** [jfrog/jfrog-idea-plugin#212]
**Is your feature request related to a problem? Please describe.**
For a particular Gradle project, the xray scans usually take like 15 mins to complete and the plugin always throws a "[ERROR] Xray Scan failed: IOException: Xray get graph scan exceeded the timeout" in about 6 mins

**Describe the solution you'd like to see**
I've been able to get the scan id from debug logs and get results from API. But that doesn't have the UI visibility I like to see.
Can we have a timeout settings for the same?

Otherwise being able to import the json result to the IDE plugin would also help.

**Describe alternatives you've considered**

**Additional context**

**Comment by yahavi** [comment]
@colourspectrum,
The connection timeout can already be configured in the `JFrog Global Configuration`:
<img width="625" alt="image" src="https://user-images.githubusercontent.com/11367982/170459188-a22456fc-fcee-4f54-a0b8-ef0be35142da.png">

Please let me know if that helped.

By the way, we are planning to introduce significant performance improvements for Gradle in the next version:
* New scan algorithm - https://github.com/jfrog/ide-plugins-common/pull/77
* Caching issue fix - https://github.com/jfrog/build-info/pull/644

So stay tuned 😎

**Comment by colourspectrum** [comment]
@yahavi this unfortunately doesn't work.
I set the timeout value to some 2000+ and it still stops polling the scan results in around 6 minutes.
Which is the same when it was set to default 60.

I do not think this configuration is relevant, because the issue is not with 'connectivity' but with scan results taking too long.
I am able to see constant debug logs where it succesfully gets pending scan result with 202 Status.

PS it's good to know that gradle improvements are planned. Because this particular issue I've encountered when 1-2 particular dependencies when used in Gradle projects, for Maven the Xray results are instanteneous.
It's that big of a deal that I need to isolate the dependencies and create a dummy Maven project to get the results.
Let me know if a sample project or a creating a new issue would be worth.
